content/06.methods.md

```diff
@@ -183,54 +183,54 @@ We filtered out any CNVs that overlapped 50% or more with immunoglobulin, telome
 ##### Gene Expression Matrices with Unique HUGO Symbols
 
 To enable downstream analyses, we next identified gene symbols that map to multiple Ensembl gene identifiers (in GENCODE v27, 212 gene symbols map to 1866 Ensembl gene identifiers), known as multi-mapped gene symbols, and ensured unique mappings (`collapse-rnaseq` analysis module in the OpenPBTA Analysis repository).
-To this end, we first removed genes with no expression from the RSEM abundance data by requiring an FPKM > 0 in at least 1 sample across the PBTA cohort.
+To this end, we first removed genes with no expression from the `RSEM` abundance data by requiring an FPKM > 0 in at least 1 sample across the PBTA cohort.
 We computed the mean FPKM across all samples per gene. 
 For each multi-mapped gene symbol, we chose the Ensembl identifier corresponding to the maximum mean FPKM, using the assumption that the gene identifier with the highest expression best represented the expression of the gene.
 After collapsing gene identifiers, 46,400 uniquely-expressed genes remained in the poly-A dataset, and 53,011 uniquely-expressed genes remained in the stranded dataset. 
 
 ##### Gene fusion detection
 
-We set up Arriba [@doi:10.1101/gr.257246.119] and STAR-Fusion [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
-For both of these tools, we used aligned BAM and chimeric SAM files from STAR as inputs and `GRCh38_gencode_v27` GTF for gene annotation.
-We ran STAR-Fusion with default parameters and annotated all fusion calls with the `GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz` file from the STAR-Fusion release.
-For Arriba, we used a blacklist file `blacklist_hg38_GRCh38_2018-11-04.tsv.gz` from the Arriba release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
-We provided Arriba with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
-We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) on Arriba fusion calls to harmonize annotations with those of STAR-Fusion.
+We set up `Arriba` [@doi:10.1101/gr.257246.119] and STAR-Fusion [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
+For both of these tools, we used aligned BAM and chimeric SAM files from `STAR` as inputs and `GRCh38_gencode_v27` GTF for gene annotation.
+We ran `STAR-Fusion` with default parameters and annotated all fusion calls with the `GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz` file from the `STAR-Fusion` release.
+For `Arriba`, we used a blacklist file `blacklist_hg38_GRCh38_2018-11-04.tsv.gz` from the `Arriba` release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
+We provided `Arriba` with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
+We used [`FusionAnnotator`](https://github.com/FusionAnnotator/FusionAnnotator) on `Arriba` fusion calls to harmonize annotations with those of `STAR-Fusion`.
```

Comment:
haha, didn't notice. I think it is linked to its own github since it is not published. Then, linked to our github for the repro workflow. Since this is forked to d3b, do you think leaving the latter is fine?